src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.FileStream.cs

```diff
@@ -135,6 +135,9 @@ public void SeekBackward(long fileSize, FileOptions options)
             }
         }
 
+        [MemoryRandomization(true)]
+        [IterationCount(50)]
+        [Outliers(Perfolizer.Mathematics.OutlierDetection.OutlierMode.DontRemove)]
```

Comment:
After my recent work on `FileStream` rewrite I doubt that any of the File IO benchmarks are affected by alignment issues.

@DrewScoggins have you tried comparing the results for MemoryRandomization enabled vs disabled for this benchmark? If we can't observe any gains, we should not enable it.

```suggestion
```